tests/validators/valid_urls.txt

```diff
@@ -25,6 +27,10 @@ https://www.example.com/foo/?bar=baz&inga=42&quux
 http://✪df.ws/123
 http://userid@example.com
 http://userid@example.com/
+http://userid@example.com:0000000000000000080
+http://userid@example.com:0000000000000000080/
```

Comment:
I don't think that we really want to support such ports, leading zeros are always tricky.